.travis.yml

```diff
@@ -2,17 +2,20 @@ language: c
 install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq curl
-  - wget http://download.opensuse.org/repositories/home:/tusavik/xUbuntu_12.04/Release.key
+  - wget http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04/Release.key
   - sudo apt-key add Release.key
-  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/tusavik/xUbuntu_12.04/ ./" >> /etc/apt/sources.list'
+  - rm -rf Release.key
+  - wget http://download.opensuse.org/repositories/home:/viktorjuhasz/xUbuntu_12.04/Release.key
+  - sudo apt-key add Release.key
+  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04 ./" >> /etc/apt/sources.list'
+  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/viktorjuhasz/xUbuntu_12.04/ ./" >> /etc/apt/sources.list'
```

Comment:
And what is the problem with additional repositories?

On Mon, Aug 17, 2015 at 4:36 PM, Budai Laszlo <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/balabit/syslog-ng-incubator/pull/140#discussion_r37193686>
> :
>
> >    - sudo apt-key add Release.key
> > -  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/tusavik/xUbuntu_12.04/ ./" >> /etc/apt/sources.list'
> > +  - rm -rf Release.key
> > +  - wget http://download.opensuse.org/repositories/home:/viktorjuhasz/xUbuntu_12.04/Release.key
> > +  - sudo apt-key add Release.key
> > +  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04 ./" >> /etc/apt/sources.list'
> > +  - sudo sh -c 'echo "deb http://download.opensuse.org/repositories/home:/viktorjuhasz/xUbuntu_12.04/ ./" >> /etc/apt/sources.list'
>
> Are you sure in that you will need both repositories?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balabit/syslog-ng-incubator/pull/140/files#r37193686>.
>